src/main/java/org/tikv/common/region/AbstractRegionStoreClient.java

```diff
@@ -81,10 +80,6 @@ protected AbstractRegionStoreClient(
     this.store = store;
     if (this.store.getProxyStore() != null) {
       this.timeout = conf.getForwardTimeout();
-    } else if (!this.store.isReachable()) {
```

Comment:
why it's removed?

Reply:
We do not need to do `Seek Leader` and `Seek Proxy` during initializing `RegionStoreClient`. Reasons are:
- `Seek Leader` and `Seek Proxy` cost time, but we do not want to the initializing process cost too much time.
- `Seek Leader` and `Seek Proxy` will happen after the first call when reciving gRPC Deadline Exceeded.